Add Openn311_ServiceRequest.

TakashiAbe0309 opened the pull request:
I wrote a gas to register the data of Open311_ServiceRequest in Orion.

TakashiAbe0309 committed:
Add Openn311_ServiceRequest.
I wrote a gas to register the data of Open311_ServiceRequest in Orion.

YujiAzama commented on Open311_ServiceRequest/src/open311_ServiceRequest.gs:
```diff
@@ -0,0 +1,67 @@
+function onEditIssues() {
+  var sheet = SpreadsheetApp.getActive().getSheetByName('Issues');
```
I think the sheet name should define the type of standard data model. (e.g. Open311ServiceRequest)

TakashiAbe0309 replied:
Thank you for the review.
I change the spreadsheet name from Issues to Open311ServiceRequest.

YujiAzama commented on Open311_ServiceRequest/src/open311_ServiceRequest.gs:
```diff
@@ -0,0 +1,67 @@
+  var splitRow = location.indexOf(',');
+  var latitude = location.slice(0, splitRow);
+  var longitude = location.slice(splitRow + 1)
+  var media_url = sheet.getRange(lastRow, 4).getValue();
```
You should use lowerCamelCase for local variables.

TakashiAbe0309 replied:
Thank you for the review. I fix the pointed-out part.

YujiAzama commented on Open311_ServiceRequest/src/open311_ServiceRequest.gs:
```diff
@@ -0,0 +1,67 @@
+  var latitude = location.slice(0, splitRow);
+  var longitude = location.slice(splitRow + 1)
+  var media_url = sheet.getRange(lastRow, 4).getValue();
+  var service_code = sheet.getRange(lastRow, 5);
```
ditto

YujiAzama commented on Open311_ServiceRequest/src/open311_ServiceRequest.gs:
```diff
@@ -0,0 +1,67 @@
+  var longitude = location.slice(splitRow + 1)
+  var media_url = sheet.getRange(lastRow, 4).getValue();
+  var service_code = sheet.getRange(lastRow, 5);
+  var service_name = sheet.getRange(lastRow, 6).getValue();
```
ditto

TakashiAbe0309 committed:
Fix sheet name and variables of Open311-ServiceRequest script.
I changed the sheet name of the script from Issues to Open311ServiceRequest.
I modified the local variable of the script to lowerCamelCase.

TakashiAbe0309 reviewed (commented):
I fixed what was pointed out in the review.
I changed the sheet name of the script from Issues to Open311ServiceRequest.
I modified the local variable of the script to lowerCamelCase.

YujiAzama reviewed (approved):
LGTM
